TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/FoundryGraphicsTestsEditor.asmdef

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Unity.ShaderGraph.Editor.GraphicsTests",
```

Comment:
This assembly name should probably be `Unity.ShaderFoundry.Editor.GraphicsTests`

Reply:
Yeah, I had it changed to ShaderFoundry, but we use one ShaderGraph internal class, that I didn't feel like copying out (PreviewResources) -- so it's currently being granted internal access via the name...  we could either copy that class out, or use asmref hackery to get access to it...